templates/submission/submissionMetadataFormTitleFields.tpl

```diff
@@ -8,6 +8,26 @@
  * Submission's metadata form title fields. To be included in any form that wants to handle
  * submission metadata.
  *}
+{if count($supportedSubmissionLocaleNames) == 1}
+	{* There is only one supported submission locale; choose it invisibly *}
+	{foreach from=$supportedSubmissionLocaleNames item=localeName key=locale}
+		<input type="hidden" name="locale" value="{$locale|escape}" />
+	{/foreach}
+{else}
+	{* There are several submission locales available; allow choice *}
+	{fbvFormSection title="submission.submit.submissionLocale" for="locale"}
+		{fbvElement type="select" label="submission.submit.submissionLocaleDescription" id="locale" from=$supportedSubmissionLocaleNames selected=$locale translate=false readonly=$readOnly size=$fbvStyles.size.MEDIUM}
+	{/fbvFormSection}
+{/if}{* count($supportedSubmissionLocaleNames) == 1 *}
```

Comment:
why is this part here, when it is already in core:submission/submissionLocale.tpl and included in controllers/modals/submissionMetadata/form/issueEntrySubmissionReviewForm.tpl just before this template?